### DISCUSSION TITLE
How to reset Hoymiles Energy Meter

### DISCUSSION THREAD

**Discussion by DennYoDeluxe** [tbnobody/OpenDTU#1136] [Q&A]
Hi there, I bought a used Hoymiles Inverter HM-600. Is there any way to reset the inverters Energy Produced Counter so I can start from zero?
Hoymiles has an own Wireless Module which connects to their cloud and with that it seems possible to reset the device.

Maybe there is way with OpenDTU, too?

regards,
Dennis

**Comment by tbnobody** [comment]
Not really a reset but maybe a suitable workaround?
https://github.com/tbnobody/OpenDTU/wiki/FAQ#q-is-it-possible-to-reset-the-yield-total-value

**Comment by ke352802081770314** [comment]
the link quoted above is no longer valid (I looked for this since I just bought a used inverter myself). 
https://www.opendtu.solar/firmware/faq/ states:

>Is it possible to reset the yield total value?

>The yield total value comes directly out of the inverter. (Currently) there is no way to reset or change this value. Please consider it as the total kilometer counter in a car. On the other hand it's possible to add a yield offset to each inverter. This can be set up in the inverter properties (Settings --> Inverters --> Click on the pen icon)

(perhaps the Hoymiles cloud does the same - only "reset" it by storing a difference outside the device, and calculate it when asked via the cloud.)
